tests/e2e/pact/broker/verify.ts

```diff
@@ -123,7 +129,54 @@ async function verifyTracker(): Promise<void> {
   await verifyOnce("tracker")
 }
 
+function clearData() {
+  if (isSandbox) return
+  const releaseUrl = `${process.env.PACT_PROVIDER_URL}${basePath}/Task/$release`
+
+  const nominatedReleases = fetcher.taskExamples
+    .filter(task => task.isSuccess)
+    .filter(task => task.requestFile.operation === "release")
+    .map(task => task.request as fhir.Parameters)
+    .filter(isNominatedRelease)
+
+  nominatedReleases.forEach(async release => {
+    let response
+    do {
+      console.log(
+        "Clearing Prescriptions For: ",
+        getIdentifierParameterByName(release.parameter, "owner").valueIdentifier.value
+      )
+      response = await nominatedRelease(releaseUrl, release)
+    }
+    while (response && response.data.resourceType !== "OperationOutcome")
+  })
+}
+
+async function nominatedRelease(releaseUrl: string, release: fhir.Parameters) {
+  return await axios.post<fhir.Bundle | fhir.OperationOutcome>(
+    releaseUrl,
+    release,
+    {
+      headers: {
+        "Content-Type": "application/fhir+json; fhirVersion=4.0",
+        "X-Request-ID": uuid.v4(),
+        "X-Correlation-ID": uuid.v4(),
+        "Authorization": `Bearer ${process.env.APIGEE_ACCESS_TOKEN}`
+      }
+    }
+  ).catch(e => console.log(e))
```

Comment:
Might be a good idea not to catch the exception here but instead put a try catch around the do while loop. That way we won't get stuck retrying forever if we get an error response from the API